modules/user/src/main/UserRepo.scala

```diff
@@ -336,18 +333,30 @@ object UserRepo {
     }
 
   def setEmail(id: ID, email: EmailAddress): Funit =
-    coll.update($id(id), $set(F.email -> email) ++ $unset(F.prevEmail)).void
+    coll.update($id(id), $set(F.email -> email.normalize) ++ $unset(F.prevEmail)).void
 
-  def email(id: ID): Fu[Option[EmailAddress]] = coll.primitiveOne[EmailAddress]($id(id), F.email)
+  def email(id: ID): Fu[Option[EmailAddress]] = coll.primitiveOne[EmailAddress]($id(id), F.email) // downgrade from normalized (#2509)
+
+  def enabledWithEmail(email: NormalizedEmailAddress): Fu[Option[(User, EmailAddress)]] =
+    coll.find($doc(
+      F.email -> email,
+      F.enabled -> true
```

Comment:
Assuming there is indexed lookup by email, will this still be used? If not, should I check `enabled` in Scala instead?

Reply:
`email` is indexed, and mongodb will also check that `enabled: true`